src/templates/uploads.html

```diff
@@ -65,10 +65,10 @@
                                       <span class='3dmol-style' data-divid='{{ moldivId }}' data-style='cross'>{{ 'Cross'|trans }}</span>
                                     </li>
                                     <li class='dropdown-item'>
-                                      <span class='3dmol-style' data-divid='{{ moldivId }}' data-style='line'>{{'Line'|trans }}</span>
+                                      <span class='3dmol-style' data-divid='{{ moldivId }}' data-style='line'>{{' Line'|trans }}</span>
                                     </li>
                                     <li class='dropdown-item'>
-                                      <span class='3dmol-style' data-divid='{{ moldivId }}' data-style='sphere'>{{'Sphere'|trans }}</span>
+                                      <span class='3dmol-style' data-divid='{{ moldivId }}' data-style='sphere'>{{' Sphere'|trans }}</span>
```

Comment:
Note: you cannot do that, as this changes the translated string and it loses all the translations!!

Reply:
Sorry, that was a mistake. The space should have been before and not within the quotation mark: `{{ 'Sphere'|trans }}` like e.g. [here](https://github.com/elabftw/elabftw/pull/2099/files/5afe69ed58bb3ff3a944b46c55714bc17e4128a0#diff-5ae35f6574894ec9281629945aa0e802R65)